src/libs/KeyboardAvoidingView/index.android.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
I don't think this is any different than `index.js`, is that correct? If there is no difference, then it's OK to remove this `android` file.